src/components/Card/TimeSelectorCard.tsx

```diff
@@ -0,0 +1,50 @@
+import { FC, useState } from 'react'
+import { ITimestamp } from '../time-selector/TimeSelectorInterfaces'
+import { timeTo12HrFormat } from '../../utils/TimeSelectorUtil'
+import './Card.css'
+
+interface ICardProps {
+    name: string
+}
+
+export const TimeSelectorCard: FC<ICardProps> = ({ name }) => {
+
+    const [startTime, setStartTime] = useState<string>('')
+    const [endTime, setEndTime] = useState<string>('')
+    const [timestamp, setTimestamp] = useState<ITimestamp[]>([])
+
+    const getTimestamps = () => {
+        return ( 
+            timestamp?.map((time, index) => {
+                const { startTime, endTime } = time
+                return <div key={index}>
```

Comment:
Please always try to use a unique value for key other than index. If we have another looping UI component in same React component then it will have problems rendering the UI.
We can append a unique string for each loop along with looping index.